.github/workflows/web3-docker-publish.yml

```diff
@@ -0,0 +1,85 @@
+name: Web3 Docker Build Push
+
+on:
+  push:
+    branches: ["develop", '*-rc*']
+    paths:
+       - 'web3/**'
+    # todo: how to release tag under monorepo?
+    tags: ["web3@v*.*.*"]
```

Comment:
We use a tag for the whole repository.
We shouldn't release submodules separately. 

For example: If we want to release features of web3. We should bump the version to the next tag number and release the whole project.